api-specificatie/zrc/openapi.yaml

```diff
@@ -0,0 +1,194 @@
+openapi: "3.0.0"
+info:
+  title: Zaak registratie API
+  description: Een API om een zaak registratie component te kunnen benaderen
+  version: 1.0.0
+  contact:
+    url:  https://github.com/VNG-Realisatie/gemma-zaken
+paths:
+  /zaken:
+    post:
+      summary: Aanmaken nieuwe zaak
+      operationId: aanmakenZaak
+      description: Maakt een nieuwe zaak aan en retourneert de toegekende zaak identificatie wanneer de zaak succesvol is aangemaakt.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Zaak'
+      responses:
+        201:
+          description: Zaak is succesvol aangemaakt.
+          content:
+            application/json:
+              schema:
+                type: string
+                description: Zaak identificatie toegekend aan de nieuwe zaak.
+          headers:
+            location:
+              description: locatie van de nieuwe zaak
+          links:
+            toevoegenInformatieobject:
+              $ref: '#/components/links/ToevoegenInformatieobject'
+        422:
+          description: Opgegeven zaak informatie kan niet verwerkt worden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        default:
+          description: Onverwachte fout tijdens opvoeren zaak
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+  /zaken/{zaak-uuid}/informatieobjecten:
```

Comment:
ik zou moeten in de informatiemodellen duiken hiervoor, maar ik *dacht* dat een informatieobjecten bij meerdere zaken kan horen (many to many), als je dan de DSO API strategie volgt, dan zou infromatieobjecte als onafhankelijke resource moeten ontsloten worden.

Reply:
Informatieobjecten kunnen inderdaad bij meerdere zaken horen. Dat wil zeggen, er kunnen meerdere zaken betrekking hebben op hetzelfde informatieobject.